### ISSUE TITLE
Add contains, equals and startsWith to matchRule

### ISSUE THREAD

**Issue by borkurdotnet** [FutureCitiesCatapult/TomboloDigitalConnector#268]
Currently the matchRule assumes that the pattern is something that can be fed into the SQL like function (e.g. name like 'Leeds%'). It would be more straight forward to provide a non-SQL specific function such as startsWith, equals and contains.

Example: Get me all subjects whose name is equal to 'Greenwich'
```
"matchRule" : {
    "attribute" : "name",
    "equals" : "Greenwich"
}
```

Example: Get me all subjects whose label starts with 'E090'
```
"matchRule" : {
    "attribute" : "label",
    "equals" : "E090"
}
```

Example: Get me all subjects whose name contains the string 'IoT'
```
"matchRule" : {
    "attribute" : "name",
    "contains" : "IoT"
}
```

**Comment by lorenaqendro** [comment]
See also #128 and #359.

**Comment by arya-hemanshu** [comment]
closed with pr #515
